test/kitchen/site-cookbooks/dd-system-probe-check/recipes/linux_use_azure_mnt.rb

```diff
@@ -0,0 +1,12 @@
+return unless azure? && !platform?('windows')
+
+mnt_path = ::File.exist?('/mnt/resource') ? '/mnt/resource' : '/mnt'
```

Comment:
Where does `/mnt/resource` come from?

Reply:
On some azure vms the temp storage is mounted there instead of `/mnt`, in particular centos